freebasics/templates/500.html

```diff
@@ -1,21 +1,14 @@
 
+
+{% extends "error_page_base.html" %}
```

Comment:
The reason `500` is different it's because usually it's a result of something horribly wrong going on, possible in the base template e.g something in the django context is failing to load, and so all the nice stuff in `base.html` will throw an error and so rendering a template that relies on the same base template is problematic. 

The only alternative is to skin it to look like consistent, but you can't inherit from `base.html`